docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Is this file needed?

Reply:
For me it's more convenient to use docker compose, instead of linking the volume manually. But I can also remove it if you want

Reply:
Is it used to have valgrind indide? Fine for keeping it 👍🏼

Reply:
yes, we can't run valgrind on macos, hence the Dockerfile and docker-compose.yml